src/Navigation.js

```diff
@@ -1,44 +1,122 @@
+        <Navbar bg="light" expand="lg">
+            <Navbar.Brand href="/">
+                <img
+                    className="d-inline-block align-top"
+                    src="public/assets/images/AnitaBLogo.png"
+                    alt="Logo"
+                />
+            </Navbar.Brand>
+            <Navbar.Toggle aria-controls="basic-navbar-nav" />
+            <Navbar.Collapse id="basic-navbar-nav">
+                
+                    
+                        <Nav className="mr-auto" id="mainNav">
+                            <Nav.Item>
+                            <Accordion>
+                                    <Card>
+                                        <Card.Header>
+                                            <Accordion.Toggle as={Link} to="/" eventKey="0">Home</Accordion.Toggle>
+                                        </Card.Header>
+                                        <Accordion.Collapse eventKey="0">
+                                            <Nav>
+                                                <Nav.Link as={Link} to="/about">About</Nav.Link>
+                                                <Nav.Link as={Link} to="/benefit">Benefit</Nav.Link>
+                                                <Nav.Link as={Link} to="/contact">Contact</Nav.Link>
+                                            </Nav>
+
+                                        </Accordion.Collapse>
+                                    </Card>
+                                </Accordion>
+                                </Nav.Item>
+
+                            <Nav.Item>
+                            <Accordion>
+                                    <Card>
+                                        <Card.Header>
+                                            <Accordion.Toggle as={Link} to="/members" eventKey="0">Members</Accordion.Toggle>
+                                        </Card.Header>
+                                    </Card>
+                                </Accordion>
+                            </Nav.Item>
+                            <Nav.Item>
+                            <Accordion>
+                                    <Card>
+                                        <Card.Header>
+                                            <Accordion.Toggle as={Link} to="my-portfolio" eventKey="0">My Space</Accordion.Toggle>
+                                        </Card.Header>
+                                        <Accordion.Collapse eventKey="0">
+                                            <Nav>
+                                                <Nav.Link as={Link} to="/portfolio">Portfolio</Nav.Link>
+                                                <Nav.Link as={Link} to="/profile">Profile</Nav.Link>
+                                                <Nav.Link as={Link} to="/request-history">Request History</Nav.Link>
+                                            </Nav>
+
```

Comment:
there is a closing tag and indentation. why so many lines?

Reply:
as above 😁